Proposed Changes to Bestiary and Powers List (#174)
DeckofAdventures/TheGame: docs/src/1_Mechanics/01_PlayerGuide_Full.md

Another creature may use a Major action to end this effect. This effects ends
automatically if the target is damaged by the creature that caused the condition.

> 12. **Invisible** All attacks made by an Invisible creature are drawn at Upper Hand. When
> making a Check to detect an Invisible creature, draw with Lower Hand. Unless otherwise
> stated, using a Power ends the Invisible condition.

Reviewer: I think this wording is slightly confusing, given that all attacks use a power. 'All attacks' makes it sound like many such attacks could be made.

It's not clear to me why we need both Invisible and Hidden. Is Invisible a state of the world, while hidden is a state of the relationship btwn two entities?

```suggestion
12. **Invisible** When
making a Check to detect an Invisible creature, draw with Lower Hand. Unless otherwise
stated, using a Power ends the Invisible condition. The attack that ends Invisibility is 
drawn at Upper Hand.
```
